app/Repositories/ScheduleRepository.php

```diff
@@ -60,7 +60,18 @@ public function showScheduleByWeek($id)
                 'day_month_id',
                 'dayMonth.id'
             )
-            ->where()
+            ->whereBetween('day', [
+                date('d', strtotime('-'.$day.' days')),
+                date('d', strtotime('+'.(6-$day).' days'))
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Repositories/ScheduleRepository.php</br>**Line**: 65</br>Comma required after last value in array declaration